test/parallel/test-fs-make-callback.js

```diff
@@ -3,37 +3,27 @@ const common = require('../common');
 const assert = require('assert');
 const fs = require('fs');
 const cbTypeError = /^TypeError: "callback" argument must be a function$/;
+const callbackThrowValues = [null, true, false, 0, 1, 'foo', /foo/, [], {}];
 
-function test(cb) {
+const { sep } = require('path');
+const warn = 'Calling an asynchronous function without callback is deprecated.';
+
+function testMakeCallback(cb) {
   return function() {
-    // fs.stat() calls makeCallback() on its second argument
-    fs.stat(__filename, cb);
+    // fs.mkdtemp() calls makeCallback() on its third argument
+    fs.mkdtemp(`${common.tmpDir}${sep}`, {}, cb);
```

Comment:
This throws an error if the test is run in isolation `node test/parallel/test-fs-make-callback.js` and `common.tmpDir` has not been created.

I think it makes sense to run `common.refreshTmpDir()` at the beginning of the test.